nixos/tests/web-apps/kanboard.nix

```diff
@@ -0,0 +1,28 @@
+import ../make-test-python.nix (
+  { pkgs, ... }:
+
+  {
+    name = "kanboard";
+    meta.maintainers = with pkgs.lib.maintainers; [ yzx9 ];
+
+    nodes = {
+      machine =
+        { ... }:
```

Comment:
```suggestion
```

You should be able to omit this

Reply:
If this is omitted, the following error will occur, so it may be required:

```
error: function 'machine' called with unexpected argument 'config'
at /home/yzx9/git/nixpkgs/nixos/tests/web-apps/kanboard.nix:10:9:
     9|       machine =
    10|         { }:
      |         ^
    11|
```

Reply:
I meant you should be able to remove the entire lambda -- as in make the definition

```nix
{
  machine = {
    # ...
  };
}
```

Reply:
Got it!